src/test/java/core/basesyntax/service/impl/WriteToFileImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WriteToFileImplTest {
+    private static final String PATH = "src/test/resources/test_writeToFile.csv";
+    private static final String INCORRECT_PATH = "src/main/resources/report.csv";
+    private WriteToFileImpl writeToFile;
+    private ReportCreatorService reportCreatorService;
```

Comment:
if you are testing writer you should not depend on anything else
otherwise your tests is not valid and it is not a unit testing